### PR TITLE
Scroll button functionality

### PR DESCRIPTION
**Pull request by jenniferchen11**
Screen Shot 2023-01-25 at 1 24 27 PM
Screen Shot 2023-01-25 at 1 25 17 PM

### CODE REVIEW

File: components/ScrollButton.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react";
+
+import CustomImage from "./CustomImage";
+
+export default function Navbar() {
```
Comment (justinyaodu) [inline components/ScrollButton.tsx]: Looks like this should be named `ScrollButton`? Although tbh I make this mistake everywhere and the name doesn't actually matter :sweat_smile:

**Review by justinyaodu — CHANGES_REQUESTED** [review]
Tiny nitpick, otherwise lgtm :heart_eyes:

File: components/ScrollButton.tsx (hunk 1 above)
Comment (jenniferchen11) [reply components/ScrollButton.tsx]: LMAO i copied and pasted from another file and forgot to change...

**Review by navidboloorian — CHANGES_REQUESTED** [review]
The image seems a little low res and the top is cut flat, can you guys make it an svg? (Can download the svg from the figma itself by right clicking on the left-hand side and saving as svg)

**Comment by jenniferchen11** [comment]
@navidboloorian For some reason the icons have to be PNGs (I tried it with the SVG and some of the properties cannot be applied to it). Could you take another look at it and let me know if the image quality is noticeably low?

**Review by navidboloorian — APPROVED** [review]
Yeah it's actually not noticeable, I fixed the image cutoff issue where the top of the circle was cut off. LGTM!

**Review by justinyaodu — APPROVED** [review]

**Pull request merged**